feeds_ui/package.json

```diff
@@ -21,7 +21,7 @@
     "start": "npm run server",
     "start:dev": "cross-env PORT=3002 react-scripts start",
     "build": "react-scripts build",
-    "test": "jest test",
+    "test": "react-scripts test",
```

Comment:
Should we keep it as jest to keep it consistent across the repo?

Reply:
I switched this back in the underlying PR as it makes merging easier to keep them as close as possible. Once it's merged, deployed and we're not using the other repo then I think we should switch to this as it requires some slightly different config.